environments/template/group_vars/template.yml

```diff
@@ -86,6 +86,8 @@ tls_https:
     bundle_name: "star.{{ base_domain }}_ca_bundle.pem"
     key_content: "{{ https_star_private_key }}"
 
```

Comment:
Could probably use a comment as to what it is. "Value for the isMemberOf attribute for IdPs that are marked as 'guest'."?